test-yolo.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+#
+# Usage:
+#
+# Run all tests against PHP 7.3:
+#
+#   ./test-yolo.py ci-drupal-8
+
+
+import os
+import pwd
+import errno
+import pprint
+import re
+import subprocess
+import random
+import signal
+import sys
+import time
+import json
+
+import requests
+import unittest
+import shlex
+
+# pass in site name as an argument to specify a Drupal 7 or Drupal 8 site
+
+siteName = sys.argv[1]
+del sys.argv[1]
+
+commandPrefix = "terminus drush "
+
+#drupal7Site = "ci-drupal-7"
+#drupal8Site = "ci-drupal-8"
+
+class EnvCacheClearTestCase(unittest.TestCase):
+    def fixtureCommand(self):
+        return "terminus env:clear-cache " + siteName + ".dev"
+
+    def getFixtureResult(self):
+        command = shlex.split(self.fixtureCommand())
+        output = subprocess.check_output(command, stderr=subprocess.STDOUT)
+        return output
+
+    def testEnvcacheClear(self):
```

Comment:
It would be good to have a comment that says what each of these tests do.

e.g. "Confirm that the test page in the filter site has a non-zero age; the, run a cache-clear command and confirm that the age is now zero."

This is more important for the more complicated tests.